backend/web/workflows/sft_fixer.py

```
@@ -0,0 +1,969 @@
+    except NotImplementedError as e:
+        return BatchValidateResponse(
+            overall_status="not_implemented",
+            valid_count=0,
+            invalid_count=len(batch_request.instance_ids),
+            results=[
+                {"instance_id": iid, "valid": False, "status": "not_implemented", "error": str(e)}
+                for iid in batch_request.instance_ids
+            ],
+            message="Batch instance validation not yet implemented for this provider",
+        )
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)